src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1469,39 +1486,37 @@ export class HtmlVideoPlayer {
                 this.renderPgs(videoElement, track, item);
                 return;
             }
-
-            if (this.requiresCustomSubtitlesElement()) {
+        }
+        import('../../scripts/settings/userSettings').then((userSettings) => {
+            if ((!itemHelper.isLocalItem(item) || track.IsExternal) && this.requiresCustomSubtitlesElement(userSettings)) {
```

Comment:
I think we can simplify `renderTracksEvents` a bit.

1. Make `renderTracksEvents` async.
2. Get the user settings at the beginning:
```js
const { userSettings } = await import('../../scripts/settings/userSettings');
```
Yes, it will be fetched for SSA/ASS and PGS, but I believe the `userSettings` module is already loaded at this point. We can even load it at the very beginning of the player module.

3. Remove unnecessary code:
```js
    import('../../scripts/settings/userSettings').then((userSettings) => {

    (!itemHelper.isLocalItem(item) || track.IsExternal) && 
```
4. Fix the indentation.